documentation/adr/0002_replace_google_sheets_with_big_query.md

```diff
@@ -0,0 +1,14 @@
+# Replace Google Sheets with Big Query as a reporting database
+Date: 20/01/2020
+
+## Status
+approved
+
+## Context
+We need to replace google sheets as a reporting database due to us approaching the spreadsheet limit.
```

Comment:
Is it worth filling in some more context here? e.g. 'We were using the Google Sheets API to write data to Google Sheets - effectively using Google Sheets as a reporting database. This had a number of disadvantages: Google Sheets have a space limit, which meant we had filled up some tables already and lost access to these datasets; we were projected to run out of space in the vacancies table in Q1 2020; accidental edits to spreadsheets were common, resulting in substantial maintenance effort; and Google Sheets formulae were more difficult to maintain and hand over than SQL queries.'